app/shared/design-system/components/primitives/auto-input.tsx

```diff
@@ -0,0 +1,205 @@
+export class AutoInput extends EventTarget {
```

Comment:
we need a better name for this, auto input is not very descriptive, maybe PointerInput?

Reply:
CSSPointerInput since its already specialized on css, unless you can decouple it

Reply:
Got an even better name: CSSValueGuestureInput

Reply:
I am also wondering if we can get the abstraction right to have just an input with a number, not specific to CSS value and have the drag gesture applied, then use the same logic to create a CSS value put etc, reusable pieces